File: Java/1011 - Esfera.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        int raio;
        double pi = 3.14159, volumeEsfera;
        
        Scanner valor = new Scanner(System.in);
        raio = valor.nextInt();
        volumeEsfera = (4/3.0) * pi * Math.pow(raio, 3.0);
        
        System.out.println(String.format("VOLUME = %.3f", volumeEsfera));
    }
}

File: C#/1011 - Esfera.cs
using System;

namespace Esfera{
    class Program{
        static void Main(string[] args){
            int raio;
            double pi = 3.14159, volumeEsfera;
            
            raio = Convert.ToInt32(Console.ReadLine());
            
            volumeEsfera = (4/3.0) * pi * Math.Pow(raio, 3);
            
            Console.WriteLine(string.Format($"VOLUME = {volumeEsfera.ToString("0.000")}"));
        }
    }
}
